source/common/ssl/context_config_impl.cc

```diff
@@ -44,5 +44,10 @@ ContextConfigImpl::ContextConfigImpl(const Json::Object& config) {
   server_name_indication_ = config.getString("sni", "");
 }
 
+ServerContextConfigImpl::ServerContextConfigImpl(const Json::Object& config)
+    : ContextConfigImpl(config) {
+  require_client_certificate_ = config.getBoolean("require_client_certificate", false);
```

Comment:
Uhm, that's what I assumed, but that doesn't follow the style in `ContextConfigImpl` above. What's the benefit of doing that?